download.py

```diff
@@ -118,13 +121,21 @@ def activities(agent, outdir, increment = 100):
         print('Wrong credentials for user {}. Skipping.'.format(username))
         return
     search = json.loads(response.get_data())
-    totalActivities = int(search['results']['totalFound'])
+    #totalActivities = int(search['results']['totalFound']) ##we no longer have a nice little batch summary, just an array of JSON activities
```

Comment:
Go ahead and delete the commented-out code; version control will keep track of it anyway.